source/GameContainer.js

```diff
@@ -86,6 +86,7 @@ export default class GameContainer extends Pixi.Container {
         }
         this.spawnWaveInterval = 0
         this.difficulty = Reference.DIFFICULTY[0]
+        game.removeChildAt(25)
```

Comment:
What's this for? Removing a child at a specific index isn't necessarily going to remove the same thing every time.